backend/src/test/java/com/htecgroup/skynest/service/impl/UserServiceImplTest.java

```diff
@@ -215,6 +212,18 @@ void when_VerifiedAndEnabledUser_disableUser_ShouldDisableUser() {
     Assertions.assertNotNull(userEntity.getDeletedOn());
   }
 
+  @Test
+  void when_WorkerUser_promoteUser_ShouldPromoteUser() {
+    UserDto userDto = UserDtoUtil.getVerified();
+    doReturn(userDto).when(userService).findUserById(any());
+    RoleDto roleManager = RoleDtoUtil.getManagerRole();
+    when(roleService.findByName(anyString())).thenReturn(roleManager);
+    userService.promoteUser(any());
```

Comment:
Usually in tests, when you call the actual method that you are testing, a real value is used. So pls change any() with something specific